src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnService.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.ef.iverksett.felles.FamilieIntegrasjonerClient
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.arbeidsfordeling.Enhet
+import no.nav.familie.kontrakter.felles.oppgave.FinnOppgaveRequest
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class OpprettOppgaveForBarnService(private val oppgaveClient: OppgaveClient,
+                                   private val iverksettingRepository: IverksettingRepository,
+                                   private val familieIntegrasjonerClient: FamilieIntegrasjonerClient) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    fun opprettOppgaveForBarnSomFyllerAar(oppgaveForBarn: OppgaveForBarn) {
+        val behandling = iverksettingRepository.hent(oppgaveForBarn.id)
```

Comment:
Hvis vi sender med ident/aktørid til forelderen så trenger vi egentlige ikke å hente opp behandlingen fra iverksett her

Reply:
Men vi trenger eksternId og behandlingstema også ? Jeg vet ikke, skal vi hente opp det og sende det over, eller skal vi plukke det opp i iverksett ?

Reply:
Ja det er vel iofs sant, vet ikke hvordan vi burde gjøre i slike tilfeller. 
Fordelen med å sende med det fra sak er att vi ikke trenger å forholde oss til en relativt svær json fra databasen, som eventuellt kan endre seg med tiden. Hva tenker @charliemidtlyng ?

Reply:
Usikker - noen hundre oppslag - vet ikke hvor voldsomt det egentlig er. Fødselsnummer _KAN_ jo ha endret seg, men det er ikke så relevant uansett - ettersom vi oversetter det til aktørid. 

Hvordan blir det å hente det ut i ef-sak da? Må joine på fagsak og fagsakEksternId for å hente ut stønadstype og ekstern fagsakId ? Det kan jo funke. Kanskje dere to kan se litt på det sammen i morgen?